extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -173,7 +180,7 @@ public function testGetLocalEnvURLDataProvider() {
 				'env' => WIKIA_ENV_DEV,
 				'forcedEnv' => null,
 				'url' => 'http://gta.wikia.com/',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "http://gta.wikia.com/" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)